.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,8 +8,12 @@ List of open PRs: https://github.com/NixOS/nixpkgs/pulls
 Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
 -->
 
-###### Motivation for this change
+###### Description of this change
```

Comment:
Don't the commit messages already describe the change?
At least a motivation for the changes is new information.

I know that a motivation is a bit redundant for stuff like package updates, but for, say, changes to `lib` or `nixos`, the motivation is important for understanding the context and why the author wants to make the changes.

Reply:
> Don't the commit messages already describe the change?

Yes, but many PRs have multiple commits and therefore a description that summarizes them all is nice to have. Also, most people automatically state why they are doing things when asked about describing, provided that they have a motivation worth stating.